src/Pages/Wallets.razor

```diff
@@ -569,19 +569,37 @@
                                     <TableHeaderCell>Outpoint</TableHeaderCell>
                                     <TableHeaderCell>Confirmations</TableHeaderCell>
                                     <TableHeaderCell>Value</TableHeaderCell>
+                                    <TableHeaderCell>Tags</TableHeaderCell>
+                                    <TableHeaderCell>Freeze</TableHeaderCell>
                                 </TableRow>
                             </TableHeader>
                             <TableBody>
-                                @if (_detailsUTXOs?.Confirmed.UTXOs != null && _detailsUTXOs.Confirmed.UTXOs.Any())
+                                @if (_detailsUTXOs.Any())
                                 {
-                                    foreach (var utxo in _detailsUTXOs?.Confirmed.UTXOs)
+                                    foreach (var (_, utxo, tags, isFrozen) in _detailsUTXOs)
                                     {
                                         <TableRow>
                                             <TableRowHeader>
                                                 <a href="@($"{Environment.GetEnvironmentVariable("MEMPOOL_ENDPOINT")}/tx/{utxo.Outpoint.Hash}#vout={utxo.Outpoint.N}")" target="_blank">@utxo.Outpoint </a>
                                             </TableRowHeader>
                                             <TableRowCell>@utxo.Confirmations</TableRowCell>
-                                            <TableRowCell>@($"{@utxo.Value} BTC")</TableRowCell>
+                                            <TableRowCell>@($"{utxo.Value} BTC")</TableRowCell>
+                                            <TableRowCell>
+                                                @if (tags.Any())
+                                                {
+                                                    foreach (var data in tags.Select((tag, i) => new {tag, i}))
+                                                    {
+                                                        <Badge Style=@($"background-color: {Colors.GetColor(data.i)}") Pill>@data.tag.Key:@data.tag.Value</Badge>
+                                                    }
+                                                }
+                                                else
+                                                {
+                                                    @("No tags")
+                                                }
```

Comment:
Wouldnt this be super repetitive? Like seeing "No tags" 100 times

Reply:
removed